### DISCUSSION TITLE
Initialize a MySQL connection in a controller with credentials without defining in config.

### DISCUSSION THREAD

**Discussion by zLupa** [adonisjs/core#1775] [Help]
In my application, i need to contact other Databases to get information and that databases credentials is in my local database, so is possible to me initialize a connection using lucid informing the credentials in a controller or something like that?

I can't set them in config because they're specific for every request that is made.

i'm searching for better ways to do that, but no success.

If isn't possible, any workaround?

**Comment by thetutlage** [comment]
Which version of `@adonisjs/lucid` are you using?

**Comment by zLupa** [comment]
@thetutlage i'm using adonis 4.0.12

**Comment by zLupa** [comment]
Ah, i read it wrong, i'm using lucid ^6.1.3 @thetutlage
